Отчеты

marselshkl2006-arch opened the pull request.

marselshkl2006-arch committed:
Delete universities.xsl.txt

marselshkl2006-arch committed:
Delete universities.xsl

marselshkl2006-arch committed:
Delete universities.xsd

marselshkl2006-arch committed:
Delete universities.xml

marselshkl2006-arch committed:
Delete universities.dtd

marselshkl2006-arch committed:
Delete Запросы xPath directory

marselshkl2006-arch committed:
Create README.md

Отчет будет готов позже

marselshkl2006-arch committed:
Add files via upload

marselshkl2006-arch committed:
Add files via upload

Отчёт в pdf преобразовывается некорректно чтобы я не делал всё съезжает Но вот odt ведёт себя нормально побывал через разные конверторы и в либер офисе пробовал только так поэтому загружаю два

mlkv1962 reviewed (changes requested):
1. Речь в задании шла хотя бы о нескольких вузах города Омска. А у вас один Омский государственный технический университет. 
2. XML-документ следует наполнить фактическими материалами. Возьмите их на сайтах вузов.
3. XML-документ должен содержать хотя бы по одной фотографии на вуз.
4. Как оформить окончательный отчет, спросите у Шевцовой.
5. Скриншоты так не делают. Есть функция PrintScreen и редактор Paint (хотя бы)

marselshkl2006-arch committed:
Delete ОКР_Отчет.odt

marselshkl2006-arch committed:
Delete ОКР_Отчет (1).pdf

marselshkl2006-arch committed:
Add files via upload

Отчёт переделан в соответствии требованиям

marselshkl2006-arch commented:
А что конкретно корректировать код исправно работает xml файлы Я тоже поправил фотографии я привёл отчёт оформлен правильно

mlkv1962 commented:
Вопросы:
1) файл xml вообще не распознается парсером (синтаксические ошибки)
2) xsl-преобразование не выполняется
3) отсутствует папка images с фотографиями
4) как я понял html-документ вы слепили вручную, а не путем xsl-преобразования
5) можете подойти ко мне с ноутбуком и продемонстрировать функциональность своего проекта

mlkv1962 commented:
скиньте мне на почту ***@***.***

вт, 2 дек. 2025 г. в 21:31, Marsel ***@***.***>:

> *marselshkl2006-arch* left a comment
> (marselshkl2006-arch/Gasanov-Marsel-PIN-252T#8)
> <https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602625486>
>
> Могу видео скинуть у меня весь проект на windows машине стационарный ПК
> скажите куда видео скинуть я скину
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602625486>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BESXHUU4EIO52AZMMHVDSCD37WWFJAVCNFSM6AAAAACNRSJC4GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMBSGYZDKNBYGY>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

mlkv1962 commented:
Видео мне не нужно. Скиньте файлы проекта. Я уж как-нибудь разберусь )))

вт, 2 дек. 2025 г. в 21:43, Малков О.Б. ***@***.***>:

> скиньте мне на почту ***@***.***
>
> вт, 2 дек. 2025 г. в 21:31, Marsel ***@***.***>:
>
>> *marselshkl2006-arch* left a comment
>> (marselshkl2006-arch/Gasanov-Marsel-PIN-252T#8)
>> <https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602625486>
>>
>> Могу видео скинуть у меня весь проект на windows машине стационарный ПК
>> скажите куда видео скинуть я скину
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602625486>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BESXHUU4EIO52AZMMHVDSCD37WWFJAVCNFSM6AAAAACNRSJC4GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMBSGYZDKNBYGY>
>> .
>> You are receiving this because your review was requested.Message ID:
>> ***@***.***
>> com>
>>
>

mlkv1962 commented:
Гасанов, моя почта есть в Контактной работе. Пришлите мне файлы проекта

вт, 2 дек. 2025 г. в 22:03, Marsel ***@***.***>:

> *marselshkl2006-arch* left a comment
> (marselshkl2006-arch/Gasanov-Marsel-PIN-252T#8)
> <https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602786947>
> [image: image.png] Пробовал послать пробовал даже сжимать говорит слишком
> большой формат
>
> вт, 2 дек. 2025 г. в 21:44, mlkv1962 ***@***.***>:
>
> > *mlkv1962* left a comment
> (marselshkl2006-arch/Gasanov-Marsel-PIN-252T#8)
> > <
> https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602690401>
>
> > скиньте мне на почту ***@***.***
> >
> > вт, 2 дек. 2025 г. в 21:31, Marsel ***@***.***>:
> >
> > > *marselshkl2006-arch* left a comment
> > > (marselshkl2006-arch/Gasanov-Marsel-PIN-252T#8)
> > > <
> >
> https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602625486>
>
> >
> > >
> > > Могу видео скинуть у меня весь проект на windows машине стационарный
> ПК
> > > скажите куда видео скинуть я скину
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602625486>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/BESXHUU4EIO52AZMMHVDSCD37WWFJAVCNFSM6AAAAACNRSJC4GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMBSGYZDKNBYGY>
>
> >
> > > .
> > > You are receiving this because your review was requested.Message ID:
> > > ***@***.***
> > > com>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602690401>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BXMZ6OP45DO2JDWAVITETPL37WXU3AVCNFSM6AAAAACNRSJC4GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMBSGY4TANBQGE>
>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***
> > com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/marselshkl2006-arch/Gasanov-Marsel-PIN-252T/pull/8#issuecomment-3602786947>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BESXHUSPRNCYABDKUY255FL37WZ5ZAVCNFSM6AAAAACNRSJC4GVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTMMBSG44DMOJUG4>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***
> com>
>

mlkv1962 commented:
Еще раз. Я попросил прислать файлы проекта мне на почту. Почта есть в Контактной работе. Поскольку у вас возникли проблемы с кодом, я хочу детально разобраться в исходных файлах и перепроверить их. Что непонятно? Так и будем переписываться до Нового года?

mlkv1962 reviewed (approved):
Хорошо. Оформляйте отчет

marselshkl2006-arch committed:
Delete ОКР_Отчет.odt

marselshkl2006-arch committed:
Delete Cкриншоты directory

marselshkl2006-arch committed:
Доработка отчёта

Отчёт доработан в соответствии образцу Юлии также загружаю два формата потому что в odt ничего не съезжает А в pdf есть сдвиги также решил оставить прошлые скриншоты потому что основные изменения которые я делал в коде когда присылал вам на последнюю проверку в основном в конце начало у всех файлов  такое же

marselshkl2006-arch commented:
P.S в коммитах  забыл указать что листинги из отчета  я убрал так как они занимают очень большую его часть

marselshkl2006-arch committed:
Delete ОКР_Отчет.odt

marselshkl2006-arch committed:
Delete ОКР_Отчет.pdf

marselshkl2006-arch committed:
Изменения в отчёте

Отчёт проведём в соответствие с вашим письмом. Олег Брониславович можно я его уже сдам пожалуйста

marselshkl2006-arch commented:
Олег Брониславович гляньте отчёт пожалуйста

mlkv1962 commented:
Отчет никуда не годится. На лекции сегодня буду объяснять

marselshkl2006-arch committed:
Delete ОКР_Отчет.pdf

marselshkl2006-arch committed:
Delete ОКР_Отчет.odt

marselshkl2006-arch committed:
Переделанный отчёт

marselshkl2006-arch committed:
Delete ОКР Отчет git.pdf

marselshkl2006-arch committed:
Форматирование отчёта было приведено в соответствии образцу
